File: backendworker/NHSOnline.Backend.Worker/GpSystems/Suppliers/Tpp/Models/PatientRecord/Item.cs
using System.Xml.Serialization;

namespace NHSOnline.Backend.Worker.GpSystems.Suppliers.Tpp.Models.PatientRecord
{
    public class Item
    {
        [XmlAttribute("id")]
        public string Id { get; set; }
        
        [XmlAttribute("description")]
        public string Description { get; set; }
        
        [XmlAttribute("date")]
        public string Date { get; set; }
        
        [XmlText]
        public string Value { get; set; }
    }
}

File: bddtests/src/main/kotlin/mocking/tpp/models/Item.kt
package mocking.tpp.models

import javax.xml.bind.annotation.*

@XmlRootElement(name = "Item")
@XmlAccessorType(XmlAccessType.FIELD)
data class Item(
        @XmlAttribute var id: String = "default patientId",
        @XmlAttribute var description: String = "default description",
        @XmlAttribute var date: String = "default date",
        @XmlValue var value: String = "default value"
)